### ISSUE TITLE
IDLE should not make About be a transient of withdrawn root window

### ISSUE THREAD

**Issue by 6c6b8f17-f925-4a4a-bb51-e683b22af1e2** [python/cpython#83063]
BPO | [38882](https://bugs.python.org/issue38882)
--- | :---
Nosy | @terryjreedy, @taleinat, @ned-deily

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = 'https://github.com/terryjreedy'
closed_at = None
created_at = <Date 2019-11-21.18:41:20.820>
labels = ['OS-mac', 'expert-IDLE', 'type-bug', '3.9']
title = 'IDLE should not make About be a transient of withdrawn root window'
updated_at = <Date 2020-01-07.22:36:02.329>
user = 'https://bugs.python.org/culler'
```

bugs.python.org fields:
```python
activity = <Date 2020-01-07.22:36:02.329>
actor = 'terry.reedy'
assignee = 'terry.reedy'
closed = False
closed_date = None
closer = None
components = ['IDLE', 'macOS']
creation = <Date 2019-11-21.18:41:20.820>
creator = 'culler'
dependencies = []
files = []
hgrepos = []
issue_num = 38882
keywords = []
message_count = 8.0
messages = ['357195', '357241', '357289', '357291', '357316', '357317', '357320', '359557']
nosy_count = 4.0
nosy_names = ['terry.reedy', 'taleinat', 'ned.deily', 'culler']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue38882'
versions = ['Python 3.9']
```

</p></details>

**Comment by ronaldoussoren** [comment]
Is this still relevant? If I read the code correctly both idlelib/macosx.py and idllib/editor.py use the regular Tk root for the AboutDialog.  The about dialog also shows correctly for me with the IDLE.app included in the 3.12 installer which uses a version of Tk newer than 8.6.10.

https://github.com/python/cpython/blob/10e9bb13b8dcaa414645b9bd10718d8f7179e82b/Lib/idlelib/macosx.py#L191-L195

https://github.com/python/cpython/blob/10e9bb13b8dcaa414645b9bd10718d8f7179e82b/Lib/idlelib/editor.py#L623-L627

**Comment by terryjreedy** [comment]
No.  OP never saw an issue on recent 3.x and neither of us do.  Thanks for checking.
